synapse/cortex.py

```diff
@@ -1448,14 +1368,16 @@ def addLayerCtor(self, name, ctor):
         await info.set('owner', owner)
         await info.set('layers', layers)
 
-        view = await View.anit(self, node)
+        view = await s_view.View.anit(self, node)
         self.views[iden] = view
 
         return view
 
     async def delView(self, iden):
         '''
         Delete a cortex view by iden.
+
+        TODO:  determine if there are any unique layers for that view and delete those too?
```

Comment:
Only if the layer guid and the view guid are the same? ( otherwise let them add/del them explicitly )

Reply:
ok